internal/peer/common/common.go

```diff
@@ -305,6 +305,13 @@ func InitCmd(cmd *cobra.Command, args []string) {
 		LogSpec: loggingSpec,
 	})
 
+	// 'peer lifecycle chaincode package' mandates does not need to init crypto
+	// For details, please refer to FAB-17453
+	if cmd != nil && strings.Contains(cmd.CommandPath(), "lifecycle") && cmd.Name() == "package" {
+		mainLogger.Debug("peer lifecycle chaincode package does not need to init crypto")
```

Comment:
Looks good to me - could you please just add a test case?

Reply:
OK. thx.
 I will try to add a test case.